.orchestra/config/components/ui/qt.yml

```diff
@@ -4,6 +4,13 @@
 #@ load("/lib/shell.lib.yml", "expand_args")
 #@ load("/lib/optimization_flavors.lib.yml", "qt_optimization_flavors")
 
+#! This package (and the ui components at large) has been deprecated
+#! If you wish to build this component additional build-time components will be needed, which for
+#! Ubuntu are:
+#! gperf libcap-dev libfontconfig1-dev libfreetype6-dev libgl-dev libgl1-mesa-dev libgles2-mesa-dev
+#! libinput-dev libmount-dev libssl-dev libx11-dev libx11-xcb-dev libxcb-glx0-dev libxcb1-dev
+#! libxext-dev libxfixes-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev libxrender-dev
+
```

Comment:
We really need to get rid of all the components we don't care about.
At least all the Qt ones plus caliban and cold-revng.

I'm wary of dropping the X stuff in case we decide to build VSCode in orchestra, that's still a decision we have to think about.

Reply:
Orthogonally, I'd say to drop everything and if we ever need to build X-related stuff we can `git checkout` the component yamls. Right now they're there making clutter and doing nothing.

Reply:
I'm cleaning up things, will ping you once you can rebase.